docs/basics/functions.md

```diff
@@ -665,6 +669,109 @@ Which would return the following:
 "metrics": []
 ```
 
+### sample_include_filter
+
+| Applies to | Description |
+| :--------- | :---------- |
+| API | only creating the sample if both a key and value is found in the sample |
+|| in absence of sample_include_filter, it defaults to include all|
+|| if the sample cannot pass sample_include_filter filter, the sample_filter and sample_exclude_filter will not be evalucated|
+
+**Example**
+
+Consider a service that returns the following payload:
+```json
+{
+  "usageInfo": [
+    {
+      "quantities": 10,
+      "customerId": "abc"
+    },
+    {
+      "quantities": 20,
+      "customerId": "xyz"
+    }
+  ]
+}
+```
+
+You could completely skip creating the output sample:
+
+```yaml
+name: example
+apis:
+    - name: someService
+      url: http://some-service.com/samples
+      sample_include_filter:
+        - name: abc
+```
+
+Which would return the following:
+
+```json
+			"metrics": [
+				{
+					"api.StatusCode": 200,
+					"customerId": "abc",
+					"event_type": "usageInfoSample",
+					"integration_name": "com.newrelic.nri-flex",
+					"integration_version": "Unknown-SNAPSHOT",
+					"quantities": 10
+				},
+      ]
+```
+
+### sample_exclude_filter
+
+| Applies to | Description |
+| :--------- | :---------- |
+| API |  works exactly same as sample_filter |
+||if sample_filter is present, both sample_filter and sample_exclude_filter  will be applied. 
+
+**Example**
+
+Consider a service that returns the following payload:
+```json
+{
+  "usageInfo": [
+    {
+      "quantities": 10,
+      "customerId": "abc"
+    },
+    {
+      "quantities": 20,
+      "customerId": "xyz"
+    }
+  ]
+}
+```
+
+You could completely skip creating the output sample:
+
+```yaml
+name: example
+apis:
+    - name: someService
+      url: http://some-service.com/samples
+      sample_exclude_filter:
+        - name: abc
+```
+
+Which would return the following:
+
+```json
```

Comment:
Please align this to the left, and maybe using only spaces. Tabs sometimes end up ruining the formatting

Reply:
Updated. thanks!